build/cockroach.rb

```diff
@@ -150,17 +149,28 @@ def install
     ENV["GOPATH"] = buildpath
     ENV["GOHOME"] = buildpath
 
-    files = Dir.glob('*')
+    files = Dir.glob("*")
     mkdir_p buildpath/"src/github.com/cockroachdb/cockroach"
     mv files, buildpath/"src/github.com/cockroachdb/cockroach"
     Language::Go.stage_deps resources, buildpath/"src"
-    
+
     # TODO(peter): We should be using the cockroach makefile so that
     # we can get the build information baked into the
     # binary. Unfortunately, this is dying with some weird dwarf
     # linker error:
     #
     #   make -C src/github.com/cockroachdb/cockroach build SKIP_BOOTSTRAP=1
+    #
+    # Something truly bizarre is going on. The error is:
+    #
+    # # github.com/cockroachdb/c-snappy
+    # cannot load DWARF output from $WORK/github.com/cockroachdb/c-snappy/_obj//_cgo_.o: decoding dwarf section info at offset 0x0: too short
+    #
+    # But that error only happens when building with make. Using go
```

Comment:
Nothing rings any bells here. It sets `CC=clang` but that should be the default anyway. It does put its own directory at the beginning of `$PATH` so I suppose there could be something weird there, but I don't have any specific ideas.